downstream/modules/automation-hub/proc-configuring-the-client-to-verify-signatures.adoc

```diff
@@ -0,0 +1,71 @@
+
+[id="configuring-the-client-to-verify-signatures]
+
+= Configuring the client to verify signatures
+
+.Prerequsites
+* To configure the client to verify signatures requires the user to have 'su'
+privileges.
+
```

Comment:
This sentence is a bit awkward, suggest changing to:

The client must have sudo privileges configured to verify signatures.